src/types/index.ts

```diff
@@ -6,6 +6,40 @@ export type ErrorContextType = {
   setError: (message: string, error?: any) => void;
 };
 
+// TODO: Question types below should be removed when available from generated files
```

Comment:
Not part of this PR but the ErrorContextType on L4 isn't used (IMO) so it should be deleted?